jigasi/rootfs/defaults/sip-communicator.properties

```diff
@@ -103,3 +103,19 @@ org.jitsi.jigasi.xmpp.acc.ALLOW_NON_SECURE=true
 # type of non-trusted certicates. In this mode your service trust in the
 # remote certificates always.
 net.java.sip.communicator.service.gui.ALWAYS_TRUST_MODE_ENABLED=true
+
+{{ if .Env.ENABLE_JIGASI_TRANSCRIBER | default "0" | toBool }}
+# Transcription config
+org.jitsi.jigasi.ENABLE_TRANSCRIPTION=true
+org.jitsi.jigasi.transcription.ENABLE_TRANSLATION=true
+org.jitsi.jigasi.transcription.DIRECTORY=/tmp/transcripts
+org.jitsi.jigasi.transcription.BASE_URL=https://{{ .Env.LETSENCRYPT_DOMAIN }}/transcripts
```

Comment:
this won't work when the setup is behind a proxy and not using the builtin let's encrypt facility. I think we may need some PUBLIC_URL setting.